VelocitySDK/Core/VLTRecordingConfig.m

```diff
@@ -25,8 +25,8 @@ - (nonnull instancetype)initWithHandshakeResponse:(VLTPBHandshakeResponse *)hand
 {
     self = [super init];
     if (self) {
-        _sampleSize        = handshakeResponse.sampleSize;
-        _captureInterval   = handshakeResponse.captureInterval;
+        _sampleSize        = 6; // handshakeResponse.sampleSize;
```

Comment:
sure?

Reply:
For now we need to have it configured `6s` and `6s`, later on we will have in controllable from API side.